chain/network/benches/graph.rs

```diff
@@ -12,8 +12,8 @@ fn build_graph(depth: usize, size: usize) -> Graph {
 
     let mut graph = Graph::new(source.clone());
 
-    for i in 0..size {
-        graph.add_edge(&source, &nodes[i]);
+    for node in nodes.iter().take(size) {
```

Comment:
```suggestion
    for node in &nodes[..size] {
```

shorter, and also asserts that size is in bounds.